File: Spring Week 5/templates/D/sim.cpp
#include <bits/stdc++.h>
using namespace std;

string sim(string line){
    // Your code here
    return ""; // Replace this line
}

int main(){
    ios_base::sync_with_stdio(false); // Just for speeding up I/O
    cin.tie(NULL);

    int cases;
    cin >> cases;
    cin.ignore();
    for(int _ = 0; _ < cases; _++){
        string line;
        getline(cin, line);
        while(line.back() < 32) line.pop_back(); // Remove trailing whitespace
        cout << sim(line) << '\n';
    }
}

File: Spring Week 5/templates/D/sim.java
import java.io.*;
import java.util.*;

public class Main {
    
    static String sim(String line){
        // Your code here
        return "";
    }

    public static void main(String[] args) throws IOException {
        BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
        int cases = Integer.parseInt(in.readLine());
        for (int i = 0; i < cases; i++) {
            String line = in.readLine();
            System.out.println(sim(line));
        }
    }
}